tests/test_xkrx_calendar.py

```diff
@@ -0,0 +1,64 @@
+from unittest import TestCase
+
+from .test_trading_calendar import NoDSTExchangeCalendarTestBase
+from .test_utils import T
+
+from trading_calendars.exchange_calendar_xkrx import XKRXExchangeCalendar
+
+
+class XKRXCalendarTestCase(NoDSTExchangeCalendarTestBase, TestCase):
+
+    answer_key_filename = 'xkrx'
+    calendar_class = XKRXExchangeCalendar
+
+    # Korea exchange is open from 9am to 3:30pm
+    MAX_SESSION_HOURS = 6.5
+    HAVE_EARLY_CLOSES = False
```

Comment:
I see there are no early closes, but are there any holidays that may or may not fall on a weekend? In which case we would need to test that those holidays are either not observed some years, or properly roll to the next weekday? I'm just trying to think of any edge cases we might want to specifically test here, other than just a "normal" year.

Reply:
I think there are at least two holidays that I can think of that _can_ occur on a weekend.

Is there anywhere we already have a test like that?

Reply:
Here is an example of holidays falling on weekends that _are_ made up: https://github.com/quantopian/trading_calendars/blob/master/tests/test_xasx_calendar.py#L42

Here is an example of holidays falling on a weekend that _are not_ made up:https://github.com/quantopian/trading_calendars/blob/master/tests/test_xhel_calendar.py#L45